src/hotspot/share/runtime/threads.cpp

```diff
@@ -1328,6 +1328,17 @@ void Threads::print_on(outputStream* st, bool print_stacks,
         p->trace_stack();
       } else {
         p->print_stack_on(st);
+        oop thread_oop = p->threadObj();
+        if (thread_oop != nullptr) {
+          if (p->is_vthread_mounted()) {
+            oop vt = p->vthread();
+            assert(vt != nullptr, "");
```

Comment:
Please provide a valid assert string.

Reply:
Thanks ! I've fixed that in: a483113a7ff381860dec3d52a45e18d8a0fdc929